infrastructure/variables.tf

```diff
@@ -77,4 +77,9 @@ variable asp_rg_override {
 variable vault_name_override {
   description = "Vault Name"
   default = ""
+}
+
+variable "strategic_policies_privateIpsFilterPattern" {
+  description = "Private IPs Filter Pattern for Policies Evaluation"
+  default     = "10\\.\\d+\\.\\d+\\.\\d+"
```

Comment:
I thought the default was \\Q10\\E

Reply:
yeah you don't actually need to quote 10 in \Q \E
I've added in the first commit because I'm used to quoting stuff in regex but then Krem raised a point and it's actually not needed